internal/util/util_test.go

```diff
@@ -244,6 +244,36 @@ func TestMountOptionsAdd(t *testing.T) {
 		})
 	}
 }
+
+func TestParseKernelRelease(t *testing.T) {
+	t.Parallel()
+
+	badReleases := []string{ "x", "5", "5.", "5.4.", "5.x-2-oops", "4.1.x-7-oh", "5.12.x" }
+	for _, release := range badReleases {
+		_, _, _, _, err := parseKernelRelease(release)
+		if err == nil {
+			t.Errorf("release \"%s\" must not be parsed successfully", release)
```

Comment:
Instead of `\"%s\"` you can use `%q` for creating quoted strings.

Reply:
Done.